ckanext/budgetviz/groupbarchart_munc/theme/public/custom.css

```diff
@@ -0,0 +1,138 @@
+.nvd3 text,
```

Comment:
Can we change file name to style.css instead of custom.css, if this file is our base style file